eventlet/support/greendns.py

```diff
@@ -179,11 +179,16 @@ def _readlines(self):
         Note that this performs disk I/O so can be blocking.
         """
         lines = []
+        if sys.version_info < (3, 4):
+            mode = 'rU'
+        else:
+            mode = 'r'
```

Comment:
in fact, 'rb' is exactly the opposite of what I want, as it will (among other things) turn off the 'universal newlines' feature.